src/main/java/edu/hawaii/its/api/service/UpdateTimestampService.java

```diff
@@ -21,18 +25,35 @@ public class UpdateTimestampService {
 
     public static final Log logger = LogFactory.getLog(MembershipService.class);
 
-    public GroupingTimestampResult update(GroupingResult groupingResult) {
+    @Autowired
+    public GroupPathService groupPathService;
+
+    public GroupingTimestampResults update(GroupingResult groupingResult) {
         if (groupingResult.getResultCode().equals("SUCCESS")) {
-            return updateLastModifiedTimestamp(groupingResult.getGroupPath());
+            List<String> groupPaths = new ArrayList<>();
+            groupPaths.add(groupingResult.getGroupPath());
+            if (groupPathService.isOwnersGroupPath(groupingResult.getGroupPath())) {
+                groupPaths.add(groupPathService.getGroupingPath(groupingResult.getGroupPath()));
+            }
```

Comment:
I think it would be very nice to put this into a GroupPathService function that returns a list of groupPaths. Also, although our current UHGroupings API does not do this for timestamps, I think a group path of tmp:testiwta:include should update the group paths tmp:testiwta and tmp:testiwta:include. Therefore this update function should not just only cover owners group path.